cluster-autoscaler/hack/update-mods.sh

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -o pipefail
+set -o nounset
+shopt -s lastpipe
+
+err_report() {
+    echo "Exited with error on line $1"
+}
+trap 'err_report $LINENO' ERR
+
+SCRIPT_NAME=$(basename "$0")
+K8S_FORK="kubernetes/kubernetes"
+# Update here when we need a new version of Kubernetes
+K8S_REV="ef7808fec57284582c7bf3fc834570e5769df83e"
```

Comment:
Can we update that automatically? Specifically in the auto-commit mode mentioned below. 
It seems quite fragile to modify a script which is being executed though. So maybe it should be move to some other file (`update-mods.k8s_rev`?)

Reply:
The script updates a file (`hack/k8s_rev`) at the end and adds it to the commit